Sources/ImmutableMappable.swift

```diff
@@ -272,7 +272,7 @@ public extension Mapper where N: ImmutableMappable {
 		#if swift(>=4.1)
 		return try JSONArray.compactMap(mapOrFail)
 		#else
-		return try JSONArray.flatMap(mapOrFail)
+		return try JSONArray.compactMap(mapOrFail)
```

Comment:
Please just remove `else` statement if it is necessary because now `if` and `else` blocks has same actions.